css/layout.css

```diff
@@ -34,7 +34,125 @@ li {
   font-size: 18px;
 }
 
-@media screen and (max-width: 767px) {
+
+@media screen and (max-width: 375px) and (orientation: portrait) {
+
+ul {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.result-row > img {
+  width: 20%;
+  }
+
+.title-column {
+  padding-left: 2px;
+  width: 25%;
+  font-size: 13px;
+  display: flex;
+  align-items: center;
+  }
+
+.picture-column {
+  width: 25%;
+  }
+
+.buy-column, .price-column {
+  display: flex;
+  align-items: center;
+  width: 10%;
+  font-size: 11px;
+  }
+
+.save {
+  width: 15%;
+  padding-left: 2px;
+  padding-right: 2px;
+  display: flex;
+  align-items: center;
+  font-size: 11px;
+  margin-left: 2px;
+  margin-right: 2px;
+  }
+
+.save-buy {
+    display: flex;
+    align-items: center;
+    font-size: 10px;
+  }
+
+.delete {
+  font-size: 10px;
+  margin: 18px 2px 17.5px 2px;
+  }
```

Comment:
make sure your code indentation remains consistent in your media queries. The closing curly brace for each rule should be at the same indentation as the selector, and every property should be indented by one from those.